front/styles/globals.css

```diff
@@ -14,12 +14,12 @@ a {
   box-sizing: border-box;
 }
 
-@media (prefers-color-scheme: dark) {
+/*@media (prefers-color-scheme: dark) {
```

Comment:
wtf?

Reply:
I turned off the theme change to dark and light, now we have only light.
If you are against it, I will uncomment.